plane/src/typed_unix_socket/client.rs

```diff
@@ -163,16 +166,21 @@ where
                                 message,
                             } => {
                                 if let Some((_, tx)) = response_map.remove(&id) {
-                                    if let Err(e) = tx.send(message) {
-                                        tracing::error!("Error sending response: {:?}", e);
+                                    if let Err(e) = tx.send(message.clone()) {
+                                        // There's no need to log the message as e is the message itself
+                                        tracing::error!(?e, "Error sending response");
                                     }
                                 } else {
-                                    tracing::error!("No sender found for response ID: {:?}", id);
+                                    tracing::error!(
+                                        ?id,
+                                        ?message,
+                                        "No sender found for response ID."
+                                    );
                                 }
                             }
                             WrappedMessage { id: None, message } => {
-                                if let Err(e) = event_tx.send(message) {
-                                    tracing::error!("Error sending event: {}", e);
+                                if let Err(e) = event_tx.send(message.clone()) {
+                                    tracing::error!(?message, "Error sending event: {}", e);
```

Comment:
message is a datadog [reserved attribute](https://docs.datadoghq.com/logs/log_configuration/attributes_naming_convention/#reserved-attributes) so we should call it something else.

Also, rather than splicing `e` into the error message, I prefer to use structured fields like this:

```rust
tracing::error!(message=?message, ?e, "Error sending event");
```